requirements/production.txt

```diff
@@ -0,0 +1 @@
+-r testing.txt
```

Comment:
can probably delete this and use base for prod for the necessary packages to run the app. then use test/local/docs etc to install the extras